src/router/views/sensors.vue

```diff
@@ -67,6 +82,36 @@ export default {
         return null
       }
     },
+    filterSensors: function() {
+      let searchValue = this.searchInputTxt.toLowerCase().replace(/\s/g, '')
+
+      // Filtering each item (MAC, name and location) based on the input value
+      let filteredMAC = this.sensors.filter(
+        (s) => s.MAC.toLowerCase().indexOf(searchValue) > -1
+      )
+      let filteredName = this.sensors.filter(
+        (s) => s.name.toLowerCase().indexOf(searchValue) > -1
+      )
+      let filteredLocation = this.sensors.filter(
+        (s) => s.location.toLowerCase().indexOf(searchValue) > -1
+      )
+
+      // Merging the results
+      let filteredSensors = filteredMAC
+        .concat(filteredName)
+        .concat(filteredLocation)
+
+      // Removing duplicated values
+      var uniquedFilteredSensors = filteredSensors.concat()
```

Comment:
Could refactor this part by using  `uniqBy` provide by `lodash`( https://lodash.com/docs/4.17.15#uniqBy). It's up to you.

Reply:
Sure, I will use uniq